public/app/core/components/Upgrade/UpgradeBox.tsx

```diff
@@ -58,48 +58,48 @@ const getUpgradeBoxStyles = (theme: GrafanaTheme2, size: ComponentSize) => {
   const fontBase = size === 'md' ? 'body' : 'bodySmall';
 
   return {
-    box: css`
-      display: flex;
-      align-items: center;
-      position: relative;
-      border-radius: ${borderRadius};
-      background: ${theme.colors.success.transparent};
-      padding: ${theme.spacing(2)};
-      color: ${theme.colors.success.text};
-      font-size: ${theme.typography[fontBase].fontSize};
-      text-align: left;
-      line-height: 16px;
-      margin: ${theme.spacing(0, 'auto', 3, 'auto')};
-      max-width: ${theme.breakpoints.values.xxl}px;
-      width: 100%;
-    `,
-    inner: css`
-      display: flex;
-      align-items: center;
-      width: 100%;
-      justify-content: space-between;
-    `,
-    text: css`
-      margin: 0;
-    `,
-    button: css`
-      background-color: ${theme.colors.success.main};
-      font-weight: ${theme.typography.fontWeightLight};
-      color: white;
+    box: css({
+      display: 'flex',
+      alignItems: 'center',
+      position: 'relative',
+      borderRadius: borderRadius,
+      background: theme.colors.success.transparent,
+      padding: theme.spacing(2),
+      color: theme.colors.success.text,
+      fontSize: theme.typography[fontBase].fontSize,
+      textAlign: 'left',
+      lineHeight: '16px',
+      margin: theme.spacing(0, 'auto', 3, 'auto'),
+      maxWidth: theme.breakpoints.values.xxl,
```

Comment:
Is this gonna work without the `px` next to it?

Reply:
pretty sure it would, but i'll be safe and put it back 😅